extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -109,10 +261,8 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
         : this.getIntersectionElementLayoutBox();
     let sizeStr = `${this.size_.width}x${this.size_.height}`;
     const rawJson = this.element.getAttribute('json');
-    const jsonParameters = rawJson ? JSON.parse(rawJson) : {};
-    const tfcd = jsonParameters['tagForChildDirectedTreatment'];
-    const adTestOn = isInManualExperiment(this.element);
-
+    this.jsonTargeting_ = rawJson ? JSON.parse(rawJson) : {};
```

Comment:
If this throws from bad json on the element, is that handled already somewhere else? Otherwise you should use tryParseJson here